src/common/Users/Data.tsx

```diff
@@ -0,0 +1,412 @@
+      props.setIsAddUserModal(true);
+    }, 200);
+  };
+  let columns = [
+    {
+      title: t('users.userId'),
+      dataIndex: 'externalId',
+      key: 'externalId',
+      render: (externalId: string) => (
+        <Components.Text copyable>{externalId}</Components.Text>
+      ),
+    },
+    {
+      title: t('users.name'),
+      dataIndex: 'name',
+      key: 'name',
+      render: (name: string) => (
+        <Components.Text copyable>{name}</Components.Text>
+      ),
+    },
+    {
+      title: t('users.province'),
+      dataIndex: 'province',
+      key: 'province',
+    },
+    {
+      title: t('users.userType'),
+      dataIndex: 'userType',
+      key: 'userType',
+    },
+    {
+      title: t('users.userLoginId'),
+      dataIndex: 'userLoginId',
+      key: 'userLoginId',
+    },
+    {
+      title: t('users.dashboard'),
+      dataIndex: '',
+      key: '',
+      render: (text: string, record: UserData) => (
+        <Button
+        id='viewDashboard'
+          onClick={() => {
+            setSelectedUser(record);
+            setIsOpenDashboardModal(true);
+          }}>
+          {t('users.view')}
+        </Button>
+      ),
+    },
+    {
+      title: t('users.attendance'),
+      dataIndex: 'attendanceAction',
+      key: 'attendanceAction',
+      render: () => <Components.Button>{t('users.view')}</Components.Button>,
+    },
+    {
+      title: t('users.action'),
+      dataIndex: 'action',
+      key: 'action',
+      render: (text: string, record: UserData) => (
+        <ActionDropdownWithRBAC
+          arrow={true}
+          menu={{ items }}
+          placement='topLeft'
+          autoAdjustOverflow={true}>
+          <Button
+          id='actionBtn'
+            onMouseEnter={() => {
+              setSelectedUser(record);
+            }}>
+            <EditOutlined />
+          </Button>
+        </ActionDropdownWithRBAC>
+      ),
+    },
+    {
+      title: t('users.status'),
+      dataIndex: 'status',
+      key: 'status',
+      render: (status: string, record: UserData) => (
+        <StatusButtonWithRBAC
+          placement='left'
+          title={t('users.confirmation')}
+          description={t('users.statusConfirmationDesc')}
+          buttontext={status}
+          okText={t('users.yes')}
+          cancelText={t('users.no')}
+          className={status}
+          onConfirm={() => {
+            onUpdateStatus(record);
+          }}></StatusButtonWithRBAC>
+      ),
+    },
+  ];
+  const items: MenuProps['items'] = [
+    {
+      key: '1',
+      label: <EditWithRBAC>{t('users.edit')}</EditWithRBAC>,
+      onClick: () => {
+        onEditUser();
+      },
+    },
+    {
+      key: '2',
+      label: (
+        <DeleteWithRBAC
+          arrow={false}
+          placement='left'
+          key='confirm'
+          className='Popconfirm-Button'
+          title={t('users.confirmation')}
+          description={`${t('deleteConfirmationDesc')} ${selectedUser.name}?`}
```

Comment:
this should be user.deleteConfirmationDesc

Reply:
Done